lib/modules/versioning/ubuntu/index.ts

```diff
@@ -1,19 +1,21 @@
 import { regEx } from '../../../util/regex';
+import DistroInfo from '../distro';
 import type { NewValueConfig, VersioningApi } from '../types';
-import {
-  getCodenameByVersion,
-  getVersionByCodename,
-  isCodename,
-} from './distribution';
 
 export const id = 'ubuntu';
 export const displayName = 'Ubuntu';
-export const urls = ['https://changelogs.ubuntu.com/meta-release'];
+export const urls = [
+  'https://changelogs.ubuntu.com/meta-release',
+  "https://debian.pages.debian.net/distro-info-data/ubuntu.csv'",
+];
 export const supportsRanges = false;
 
 // #12509
 const temporarilyUnstable = ['22.04'];
 
+const ubuntuJsonKey = 'data/ubuntu-distro-info.json';
```

Comment:
move to distro

Reply:
I already have a string type in `distro` with this key.
other than that, how would u suggest instantiating `DistroInto` than ?

Reply:
moving those const to distro and create the combined type of this const.